src/GLM.jl

```diff
@@ -98,24 +98,24 @@ module GLM
     end
 
     const COMMON_FIT_KWARGS_DOCS = """
-        - `dropcollinear::Bool=true`: Controls whether or not a model matrix
+        - `dropcollinear::Bool`: Controls whether or not a model matrix
           less-than-full rank is accepted.
           If `true` (the default) the coefficient for redundant linearly dependent columns is
           `0.0` and all associated statistics are set to `NaN`.
           Typically from a set of linearly-dependent columns the last ones are identified as redundant
           (however, the exact selection of columns identified as redundant is not guaranteed).
-        - `method::Symbol=:cholesky`: Controls which decomposition method to use.
-          If `method=:cholesky` (the default), then the `Cholesky` decomposition method will be used.
-          If `method=:qr`, then the `QR` decomposition method (which is more stable
-          but slower) will be used.
-        - `wts::Vector=similar(y,0)`: Prior frequency (a.k.a. case) weights of observations.
+        - `method::Symbol`: Controls which decomposition method to use.
+          If `method=:qr` (the default), then the `QR` decomposition method will be used.
+          If `method=:cholesky`, then the `Cholesky` decomposition method (which is faster
+          but less accurate) will be used.
```

Comment:
Could be worth giving some more color on the tradeoffs here, but maybe too much info? Idk
```suggestion
          If `method=:cholesky`, then the `Cholesky` decomposition method will be used.
          The Cholesky decomposition is faster and more computationally efficient than
          QR, but is less numerically stable and thus may fail or produce less accurate
          estimates for some models.
```